app/views/va_facilities/index.html.erb

```diff
@@ -0,0 +1,60 @@
+<% provide :main_classes, 'bg-gray-2' %>
+<% provide :footer_classes, 'bg-gray-2' %>
+
+<% provide :head_tags do %>
+  <%= javascript_include_tag 'facility_directory', 'data-turbolinks-track': 'reload' %>
+<% end %>
+
+<section class="usa-section padding-y-0 grid-col-12">
+  <div class="grid-container position-relative grid-col-12">
+    <div class="grid-row grid-gap">
+      <div class="grid-col-12">
+        <h1 class="font-sans-2x1 margin-top-5 margin-bottom-3">
+          Facility directory
+        </h1>
+        <div class="text-base">
+          Looking for a full list of VISNs? <a href="/visns">Click here</a>
+        </div>
+        <div class="margin-top-5 margin-bottom-3">
+          <table class="usa-table--borderless">
```

Comment:
I just noticed... these filters shouldn't be in a table... they should be styled to fit in a row